cmd/brew-test-bot.rb

```diff
@@ -710,6 +710,11 @@ def homebrew
       return if @skip_homebrew
 
       if !@tap && (@formulae.empty? || @test_default_formula)
+        test "brew", "man"
+        test "git", "diff", "--exit-code", "-C", HOMEBREW_REPOSITORY, "docs/brew.1.html"
+        test "git", "diff", "--exit-code", "-C", HOMEBREW_REPOSITORY, "manpages/brew.1"
+        test "git", "diff", "--exit-code", "-C", HOMEBREW_REPOSITORY, "manpages/brew-cask.1"
```

Comment:
Interesting...call `git diff --exit-code` from `brew man` and return that exit code...that would be a nice place to put it.